### PR TITLE
preview-site should now work properly

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: docs/build-docs.sh [hunk 1]
```diff
@@ -8,8 +8,9 @@
 set -ex
 
 # Update this array with all versions of GlooMesh to include in the versioned docs website.
-readarray -t versions < <(jq -r '.versions[]' version.json)
-latestVersion=$(jq -r '.latest' version.json)
+# Update this array with all versions of Gloo Edge to include in the versioned docs website.
```
Comment (ashleywang1) [inline docs/build-docs.sh:11]:
```suggestion
```

File: docs/build-docs.sh [hunk 2]
```diff
@@ -8,8 +8,9 @@
 set -ex
 
 # Update this array with all versions of GlooMesh to include in the versioned docs website.
```
Comment (ashleywang1) [inline docs/build-docs.sh:10]:
```suggestion
# Update this array with all versions of Gloo Mesh to include in the versioned docs website.
```

File: changelog/v1.1.0-beta16/preview-docs.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+changelog:
+  - type: NON_USER_FACING
+    description: Preview docs site should not build from current branch.
```
Comment (ashleywang1) [inline changelog/v1.1.0-beta16/preview-docs.yaml:3]: Don't we want the site to build from the current PR branch?

Comment (EItanya) [reply changelog/v1.1.0-beta16/preview-docs.yaml:3]: Mistype

**Review by ashleywang1 — APPROVED** [review]

**Pull request merged**